src/core/tsi/test_creds/README

```diff
@@ -19,44 +19,68 @@ Valid test credentials:
 The ca is self-signed:
 ----------------------
 
-$ openssl req -x509 -new -newkey rsa:1024 -nodes -out ca.pem -config ca-openssl.cnf -days 3650 -extensions v3_req
+$ openssl req -x509 -new -newkey rsa:2048 -nodes -keyout ca.key -out ca.pem \
+  -config ca-openssl.cnf -days 3650 -extensions v3_req
 When prompted for certificate information, everything is default.
 
 client is issued by CA:
 -----------------------
 
-$ openssl genrsa -out client.key.rsa 1024
+$ openssl genrsa -out client.key.rsa 2048
 $ openssl pkcs8 -topk8 -in client.key.rsa -out client.key -nocrypt
-$ rm client.key.rsa
 $ openssl req -new -key client.key -out client.csr
 
 When prompted for certificate information, everything is default except the
 common name which is set to testclient.
 
-$ openssl ca -in client.csr -out client.pem
+$ openssl x509 -req -CA ca.pem -CAkey ca.key -CAcreateserial -in client.csr \
+  -out client.pem -days 3650
 
 server0 is issued by CA:
 ------------------------
 
-$ openssl genrsa -out server0.key.rsa 1024
+$ openssl genrsa -out server0.key.rsa 2048
 $ openssl pkcs8 -topk8 -in server0.key.rsa -out server0.key -nocrypt
-$ rm server0.key.rsa
 $ openssl req -new -key server0.key -out server0.csr
 
 When prompted for certificate information, everything is default except the
 common name which is set to *.test.google.com.au.
 
-$ openssl ca -in server0.csr -out server0.pem
+$ openssl x509 -req -CA ca.pem -CAkey ca.key -CAcreateserial -in server0.csr \
+  -out server0.pem -days 3650
 
 server1 is issued by CA with a special config for subject alternative names:
 ----------------------------------------------------------------------------
 
-$ openssl genrsa -out server1.key.rsa 1024
+$ openssl genrsa -out server1.key.rsa 2048
 $ openssl pkcs8 -topk8 -in server1.key.rsa -out server1.key -nocrypt
-$ rm server1.key.rsa
 $ openssl req -new -key server1.key -out server1.csr -config server1-openssl.cnf
 
 When prompted for certificate information, everything is default except the
 common name which is set to *.test.google.com.
 
-$ openssl ca -in server1.csr -out server1.pem
+$ openssl x509 -req -CA ca.pem -CAkey ca.key -CAcreateserial -in server1.csr \
+  -out server1.pem -extensions req_ext -extfile server1-openssl.cnf -days 3650
+
+Clean up:
+---------
+$ rm *.rsa
+$ rm *.csr
+$ rm ca.srl
+
+Sync up with other repositories
+===============================
+
+You will need to copy the test credentials to the following places
```

Comment:
I think list is impossible to keep up-to-date so I'm not sure if it's useful to add it here.
(if we do, it will be out of date very soon). 
Add a note that this list is incomplete and that these are just some of the locations we have to check?

Reply:
I checked all grpc repos. I think we should give a try to upgrade. 1024-bit keys and credentials would cause more and more trouble in the future. These credentials have expiration. We need to learn how to upgrade in any case.

Even some other repos use a copy of the old keys, it is likely they do not have interop issues (with gRPC repo).

Reply:
Sure, I absolutely believe you've done a good job searching through the existing repos - I'm just saying that providing the list of locations in an .md file as a documentation doesn't make much sense, because the list of locations can change any time and it is unrealistic to expect that people will update this list over time. So it's better to just say - "copies of these keys exist in multiple locations across all the grpc repos, so you need to be careful when updating the keys" (and perhaps provide some locations as examples, rather than a definitive list).

Reply:
Agree. README updated. The reason I listed the locations is because it took me a while to find all copies. It will save time for whoever does the key update next time. Hopefully, that is after 10 years when keys are expired.